tests/regression_tests/deplete_no_transport/test.py

```diff
@@ -0,0 +1,115 @@
+""" Transport-free depletion test suite """
+
+from math import floor
+import shutil
+from pathlib import Path
+
+from difflib import unified_diff
+import numpy as np
+import pytest
+import openmc
+from openmc.data import JOULE_PER_EV
+import openmc.deplete
+from openmc.deplete import FluxDepletionOperator
+
+
+from tests.regression_tests import config
+
+
+@pytest.fixture(scope="module")
+def fuel():
+    fuel = openmc.Material(name="uo2")
+    fuel.add_element("U", 1, percent_type="ao", enrichment=4.25)
+    fuel.add_element("O", 2)
+    fuel.set_density("g/cc", 10.4)
+    fuel.depletable=True
+
+    fuel.volume = np.pi * 0.42 ** 2
+
+    return fuel
+
+
+@pytest.mark.parametrize("multiproc, from_nuclides, normalization_mode, power, flux", [
+    (True, True,'constant-flux', None, 1164719970082145.0),
+    (False, True, 'constant-flux', None, 1164719970082145.0),
+    (True, True, 'constant-power', 174, None),
+    (False, True, 'constant-power', 174, None),
+    (True, False,'constant-flux', None, 1164719970082145.0),
+    (False, False, 'constant-flux', None, 1164719970082145.0),
+    (True, False, 'constant-power', 174, None),
+    (False, False, 'constant-power', 174, None)])
+def test_no_transport_from_nuclides(run_in_tmpdir, fuel, multiproc, from_nuclides, normalization_mode, power, flux):
+    """Transport free system test suite.
+
+    Runs an OpenMC transport-free depletion calculation and verifies
+    that the outputs match a reference file.
+
+    """
+    # Create operator
+    micro_xs_file = Path(__file__).parents[2] / 'micro_xs_simple.csv'
+    micro_xs = FluxDepletionOperator.create_micro_xs_from_csv(micro_xs_file)
+    chain_file = Path(__file__).parents[2] / 'chain_simple.xml'
+
+    if from_nuclides:
+        nuclides = {}
+        for nuc, dens in fuel.get_nuclide_atom_densities().items():
+            nuclides[nuc] = dens * 1e24
```

Comment:
```suggestion
        for nuc, atom_per_b_cm in fuel.get_nuclide_atom_densities().items():
            nuclides[nuc] = atom_per_b_cm * 1e24
```

Reply:
This also makes me wonder if we should just have `from_nuclides` take densities in atom/b-cm rather than atom/cm³

Reply:
I think it's normal for people to have nuclide concentrations in units other than atom/b-cm. Perhaps we should add a unit specifier to `from_nuclides`?

Reply:
I've added such a specifier